configadmin/src/main/java/org/apache/felix/cm/file/ConfigurationHandler.java

```diff
@@ -547,6 +547,9 @@ private String readQuoted( PushbackReader pr ) throws IOException
                                 buf.append( ( char ) c );
                             }
                             break;
+//                        case '\\':
+//                            buf.append("\\\\");
+//                            break;
```

Comment:
There is a copy of `ConfigurationHandler` in the utils project as well. Might be worth adding this fix to there as well so file install (and other clients) stays in sync with config admin.